paddle/operators/softmax_op.cu

```diff
@@ -12,7 +12,6 @@
    See the License for the specific language governing permissions and
    limitations under the License. */
 
-#define EIGEN_USE_GPU
```

Comment:
Is `EIGEN_USE_GPU` needed when calling `REGISTER_OP_GPU_KERNEL` to register kernel for the current op?

It's described in `new_op_cn.md`, are you intend to remove it when the operator is reusing kernels from the other operator?

Reply:
Now there is no Eigen function in `softmax_op.h`, since this op uses the library `math/softmax`. Then the macro `EIGEN_USE_GPU` can be removed.

Reply:
Thanks, I see!